bots/make-checkout

```diff
@@ -74,18 +75,35 @@ def main():
     # won't be tied to the ref anymore, and it won't exist locally, so exit
     # silently
     output = None
-    try:
-        args = ["git", "fetch", "test" if api.repo != "cockpit-project/cockpit" else "origin", opts.ref]
+    # For stable branch do not checkout refs, just change branch
+    if not opts.stable:
+        try:
+            args = ["git", "fetch", "test" if api.repo != "cockpit-project/cockpit" else "origin", opts.ref]
```

Comment:
I'm not sure why this wouldn't work when testing e. g. cockpit's rhel-8.1 branch. But this whole "test" remote is making things unnecessarily complicated, and should go away. @Gundersanne is working on that in PR #12426.